### ISSUE TITLE
Some icons using double-width glyphs look not full width

### ISSUE THREAD

**Issue by tgz033** [LunarVim/LunarVim#4013]
### Problem description

At the moment there are [three major different Nerd Fonts variants](https://github.com/ryanoasis/nerd-fonts/discussions/1103): 

- `Nerd Font Mono` is a strictly monospaced variant, with all icons and letters having the same width. However, single-width glyphs look a little bit too small for someone (like me). I guess this variant is the default choice for users to make sure icons look correct. 
- `Nerd Font` offers double-width glyphs and maintains some features of monospaced fonts. However, an extra space is required after symbols to avoid cut-off or overlapped glyphs. 
- `Nerd Font Propo` is less commonly used. 

If I use the 'JetBrainsMono Nerd Font' without the 'Mono' suffix, some of the icons may not appear at full width. However, if I add space in `lunarvim/lvim/lua/lvim/icons.lua` to display the incorrect icons, the issue will be resolved. 

I am not sure if there are other places that need to be modified or if modifying this file is a reasonable solution. 

### LunarVim version

release-1.2/neovim-0.8-8f4a7bd, and master-2d37303

### Neovim version (>= 0.8.0)

NVIM v0.8.3

### Terminal name

Windows Terminal

### Operating system/version

Windows 10 22H2

### Screenshots

![Snipaste_2023-04-08_14-59-11](https://user-images.githubusercontent.com/89851096/230718910-4c728115-77e6-4ed2-9c30-7fb7baf480ab.png)

**Comment by kobzar** [comment]
Try RUn Lvim on windows on different terminal app like alacritty for test and advice us about result.
(dont forget to setup Nerd Fonts in alacritty/kitty app first)

**Comment by tgz033** [comment]
Thanks for your reply! Following your guide, I tried to run LunarVim on *Alacritty*. The result is different from *Windows Terminal*. Here is the result on *Alacritty* using 'JetBrainsMono Nerd Font':

![Snipaste_01](https://user-images.githubusercontent.com/89851096/230768970-0f74e839-8700-432d-88f5-7abc7feb331d.png)

The right arrows look good, but the plus symbols still have a small issue. The plus symbols seem to slightly overlap the letter after them. So on *Alacritty* I tried to use 'Iosevka Nerd Font', a font that has small width of each character, to verify my guess. Here is the result:

![Snipaste_02](https://user-images.githubusercontent.com/89851096/230774129-a9f2a252-e3dc-4327-939f-bdb8c0d79e0f.png)
